Neos.Flow/Classes/Persistence/Doctrine/Mapping/Driver/FlowAnnotationDriver.php

```diff
@@ -598,6 +598,7 @@ protected function evaluatePropertyAnnotations(ORM\ClassMetadataInfo $metadata)
             $mapping['fieldName'] = $property->getName();
             $mapping['columnName'] = strtolower($property->getName());
             $mapping['targetEntity'] = $propertyMetaData['type'];
+            $mapping['nullable'] = $propertyMetaData['nullable'];
```

Comment:
Is this related to the other changes or is it a separate PR?

Reply:
It was the logical consequence of exposing the `nulllable` information in the ClassSchema, since this is the place we have it for primarily. But it could (and maybe should) go into an own PR that highlights this as an actual feature with a description on how it works/behaves.